google/resource_container_cluster_test.go

```diff
@@ -921,6 +961,30 @@ resource "google_container_cluster" "with_master_auth" {
 	}
 }`, acctest.RandString(10))
 
+var testAccContainerCluster_withNetworkPolicyEnabled = fmt.Sprintf(`
+	resource "google_container_cluster" "with_network_policy_enabled" {
+		name = "cluster-test-%s"
+		zone = "us-central1-a"
+		initial_node_count = 1
+	
+		network_policy {
+			enabled = true
+			provider = "CALICO"
+		}	
+	}`, acctest.RandString(10))
+
+var testAccContainerCluster_updateNetworkPolicyEnabled = fmt.Sprintf(`
+		resource "google_container_cluster" "with_network_policy_enabled" {
+			name = "cluster-test-%s"
+			zone = "us-central1-a"
+			initial_node_count = 1
+		
+			// remove network_policy is equal than enabled=false
```

Comment:
Instead of this comment, what about renaming the test var to `testAccContainerCluster_removeNetworkPolicy`? Right now it's a bit hard to understand.